.circleci/config.base.yml

```diff
@@ -13,17 +13,17 @@ executors:
       shell: bash.exe
     working_directory: ~/repo
     environment:
-      AMPLIFY_DIR: C:/home/circleci/repo/out
-      AMPLIFY_PATH: C:/home/circleci/repo/out/amplify.exe
+      AMPLIFY_DIR: C:/home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin
+      AMPLIFY_PATH: C:/home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin/amplify
```

Comment:
These paths are copy-paste from CLI; since API doesn't use packaged binary - this is being updated to the correct path